widgetastic_manageiq/__init__.py

```diff
@@ -4547,6 +4547,32 @@ def data_for_timestamp(self, timestamp):
         return tooltip_data
 
 
+class ReactTextInput(TextInput):
+
+    # The clear method from the WebElement class in the Selenium package was not clearing the react
+    # text field. arguments[0] is the widget name
+    clear_text_field = """\
+        (function(elem){
+            var elem_name = document.getElementsByName(elem);
+            var elem_id = elem_name[0].id;
+            document.getElementById(elem_id).value = "";
+        }(arguments[0]));
```

Comment:
I am not stuck on this current implementation. If someone had a better way to do this, will be happy to see!

Reply:
If you like; its one of good hack over `TextInput` or `TextArea`

```
In [12]: from selenium.webdriver.common.keys import Keys  
In [13]: el.send_keys(Keys.CONTROL + "a")                 
In [14]: el.send_keys(Keys.DELETE)
```
only better thing; It will remove unique widget dependency.